esx_service/cli/vmdkops_admin.py

```diff
@@ -121,6 +121,9 @@ def commands():
                                 'attached-to', 'policy', 'capacity', 'used',
                                 'fstype', 'access', 'attach-as'],
                     'metavar': 'Col1,Col2,...'
+                },
+                '--tenant' : {
+                    'help': 'Displays only VMs for a given tenant'
```

Comment:
Are there any entities to display besides VMs, if not then can reword as "Displays VMs for a given tenant".